internal_ws/ykbh/src/lib.rs

```diff
@@ -231,13 +231,13 @@ impl SIRInterpreter {
         &self.frames.last().unwrap().mem
     }
 
-    /// Inserts a pointer to the trace inputs into the `interp_step` frame.
-    pub fn set_trace_inputs(&mut self, tio: *mut u8) {
-        // The trace inputs live in $1
+    /// Inserts a pointer to the trace context into the `interp_step` frame.
+    pub fn set_interp_ctx(&mut self, ctx: *mut u8) {
+        // The trace context live in $1
```

Comment:
Not your error, but live**s*.

Reply:
Fixed in https://github.com/softdevteam/yk/pull/222/commits/f206fae132f08588599ae4d7ad538ea0c096c091.